lib/task-queue/Storage.js

```diff
@@ -1,23 +1,24 @@
-/* globals XPCOMUtils, OS, Sqlite, Task */
+/* globals XPCOMUtils, Sqlite, Task */
 "use strict";
 
 const {Cu} = require("chrome");
 
 Cu.import("resource://gre/modules/XPCOMUtils.jsm");
-Cu.import("resource://gre/modules/osfile.jsm");
 Cu.import("resource://gre/modules/Task.jsm");
 
+const {TextDecoder, TextEncoder, OS} = Cu.import("resource://gre/modules/osfile.jsm", {});
+
 XPCOMUtils.defineLazyModuleGetter(this, "Sqlite",
                                   "resource://gre/modules/Sqlite.jsm");
 
-const DB_PATH = OS.Path.join(OS.Constants.Path.profileDir, "taskqueue.sqlite");
+const DB_PATH = "taskqueue.sqlite";
```

Comment:
I found out SQLite.jsm uses the profile directory by default if we use a relative path. 
I made the change only to make the code more readable. Do you think I shouldn't?